### DISCUSSION TITLE
Is it possible to unpack a slice/array variable into a class?

### DISCUSSION THREAD

**Discussion by connor-hilll** [a-h/templ#651] [Q&A]
I have a dynamic amount of class values that get added to a slice within my handler which gets passed to the templ function through props. I was curious if anyone figured out a good way of either unpacking (which I don't think is possible) the slice as class variables

**Comment by AlexanderArvidsson** [comment]
The `class` property takes in many types: `string`, `[]string`, `[]any`, `templ.CSSClasses`, `templ.KeyValue`. You should just be able to pass that in?
https://templ.guide/syntax-and-usage/css-style-management/#dynamic-class-names

If you instead mean dynamic attributes, you can use spreading:
https://templ.guide/syntax-and-usage/attributes/#spread-attributes

If that's not what you mean, you're gonna have to be more clear, add some examples because I doubt anyone will figure out what you mean with this little information.
